InventoryManagement/src/main/java/com/ccs/inventorymanagement/service/ClothingService.java

```diff
@@ -44,7 +44,11 @@ public Mono<Clothing> update(Clothing item) {
     }
 
     @Override
-    public Mono<Clothing> delete(UUID id) {
-        return null;
+    public Mono<Clothing> delete(Clothing item) {
+        return clothingRepository.save(ClothingEntity.from(item)
+                        .toBuilder()
+                        .updated(Instant.now())
+                        .build())
+                .map(Clothing::from);
```

Comment:
A delete call should not be passing you an entity to save, you should just receive the ID of the entity requesting to be removed and then looking it up, and if it exists, perform your delete logic.